src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -675,30 +686,18 @@ private DeconstructionVariable BindDeconstructionDeclarationLocals(TypeSyntax ty
         }
 
         /// <summary>
-        /// Prepares locals corresponding to the variables of the declaration.
-        /// The locals are kept in a tree which captures the nesting of variables.
-        /// Each local is either a simple local (when its type is known) or a deconstruction local pending inference.
-        /// The caller is responsible for releasing the nested ArrayBuilders.
-        /// </summary>
-        private ArrayBuilder<DeconstructionVariable> BindDeconstructionDeclarationLocals(VariableComponentAssignmentSyntax node, TypeSyntax closestTypeSyntax, DiagnosticBag diagnostics)
-        {
-            var deconstructionVariable = BindDeconstructionDeclarationLocals(node.VariableComponent, diagnostics);
-            Debug.Assert(deconstructionVariable.HasNestedVariables);
-            return deconstructionVariable.NestedVariables;
-        }
-
-        /// <summary>
-        /// Returns a BoundLocal when the type was explicit, otherwise returns a DeconstructionLocalPendingInference.
+        /// In embedded statements, returns a BoundLocal when the type was explicit.
+        /// In global statements, returns a BoundFieldAccess when the type was explicit.
+        /// Otherwise returns a DeconstructionLocalPendingInference when the type is implicit.
```

Comment:
>DeconstructionLocalPendingInference [](start = 32, length = 35)

It looks like the rename for "DeconstructionLocalPendingInference" still hasn't happened.